system/web/context/RequestContext.cfc

```diff
@@ -1241,7 +1241,7 @@ component serializable=false accessors="true"{
 		}
 
 		//  Set content headers
```

Comment:
Can you add the following check 
```
// detect if extension is already included in name argument, if not add
		if ( listlast( arguments.name,"." ) != arguments.extension ) {
			arguments.name = arguments.name =& ".#arguments.extension#";
		}
```

Reply:
Done. See above -  used an append instead of a reset of the variable